### ISSUE TITLE
Chat UI/UX Design

### ISSUE THREAD

**Issue by TheMorpheus407** [Bootstrap-Academy/Bootstrap-Academy#145]
### Bootstrap Academy username

Morpheus

### Description

Create a mockup (no implementation) for a Chat Integration with other users directly (private messages), everyone, a team, ....
It should be visible from everywhere on the platform as a sort of overlay yet still integrate smoothly.
Important consideration: Don't clutter the website. It should probably just be one bar, that can be closed until a new message arrives? 
Ideas can be collected here.
We will need to iterate over the design multiple times probably, until we find a good solution.

**Comment by LadonFire** [comment]
Where should I submit the mockup? Should I just attach it?

**Comment by Defelo** [comment]
Yes, just post it as a comment here

**Comment by LadonFire** [comment]
Ok, I have exported the mockup as PNG files and attached them. If You need a different format, please feel free to contact me. 
Also, here is a link to the mockup: 
https://www.canva.com/design/DAGU5VFgNVE/aRj8SayZY1c6fL_394LwWA/view?utm_content=DAGU5VFgNVE&utm_campaign=designshare&utm_medium=link&utm_source=editor

[Mockup Chat UI.zip](https://github.com/user-attachments/files/17561823/Mockup.Chat.UI.zip)

**Comment by TheMorpheus407** [comment]
Thanks a ton! Looks great already :)
In the zip files, the icons and images are not really usable. Would you mind sending me a copy of the canva slides?
So we can also manipulate the images, move icons and changes borders etc?
Thanks!

**Comment by LadonFire** [comment]
Here is a link to the an edible mockup (I tried to export the slides as pptx, but this didn't worked properly): 
https://www.canva.com/design/DAGVEEui_2M/Xa2zc61AZAgduQVOBt0rhg/edit?utm_content=DAGVEEui_2M&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

Icons: 
[icon svg's.zip](https://github.com/user-attachments/files/17579013/icon.svg.s.zip)

**Comment by TheMorpheus407** [comment]
Many many thanks! 
I copied the Design and the Icons and we'll work on a second iteration based on your intial mockup :)
You can send the invoice to me and I'll assign myself on the issue, so we can keep track on every step :)

**Comment by LadonFire** [comment]
I have a question: As I'm not yet 18 years old, I can't register a business and therefore can't send invoices. This is not a problem for me, but I would like to have this certificate that you mentioned in the video in which the hackathon was announced. Would it be possible for me to get it without the invoice?

**Comment by TheMorpheus407** [comment]
Absolutely! Feel free to send me a Mail, so we can make sure, your name stays private 😊

**Comment by LadonFire** [comment]
4e)|RgqqI/f€1lh§!oWo§qwsfpN`FVhE81Xt$~6´LE^0Q
